### PR TITLE
Message card UI

### PR DESCRIPTION
**Pull request by jcswift23**
Copy over our MessageCard SwiftUI view from Planetary and modify it to display kind: 1 Nostr events.
https://github.com/planetary-social/nos/issues/5
HomeFeed

ThreadViewScreen

### CODE REVIEW

File: Nos/Views/AppView.swift [hunk 1]
```diff
@@ -7,22 +7,51 @@
...
-                NavigationLink(Localized.relaysLinkTitle.string) {
+                case .relays:
                     RelayView()
-                }
-                NavigationLink(Localized.settingsLinkTitle.string) {
+                case .settings:
                     SettingsView()
                 }
+            })
+            .navigationDestination(for: Event.self) { note in
```
Comment (mplorentz) [inline Nos/Views/AppView.swift:53]: I'm confused why we couldn't put this code in HomeFeedView.swift, but I tried moving it there and it broke the navigation 🤷‍♂️

**Review by mplorentz — COMMENTED** [review]
This is looking great! I put this in our private chat but I'll put it here for posterity. I think if we bring over `MessageButton` from Planetary we can get rid of the white borders around the cards. Feel free to do that in this PR or another one!

File: Nos/Views/AppView.swift (hunk 1 above)
Comment (jcswift23) [reply Nos/Views/AppView.swift:54]: I noticed that too, it's like it wants everything where you first define the NavigationStack.

Comment (mplorentz) [reply Nos/Views/AppView.swift:54]: Yeah I thought in the WWDC video I watched they were putting these deeper in the view heirarchy. Something to look into later I think. It's not necessary for this PR.

**Review by shanecb — APPROVED** [review]

**Pull request merged**